### PR TITLE
feat: added Project Euler Problem 13

### PR DESCRIPTION
**Pull request by byt3h3ad**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by appgurueu** [comment]
Your checks are failing.

**Comment by byt3h3ad** [comment]
all errors have been fixed

**Comment by byt3h3ad** [comment]
@appgurueu @raklaptudirm any changes needed?

**Comment by appgurueu** [comment]
I don't think this algorithm is correct in the general case. The numbers given are well below what JavaScript numbers (64-bit floats) can safely represent; that is, they exceed `MAX_SAFE_INTEGER` by a lot and get truncated to the first `n` of their most significant digits. When you now add all these numbers, you have some error. Can you rule out the possibility that this error might cause the results to be wrong? For the given test case it might work, but does it work in general? Even if it did, it wouldn't be apparent why (and would be deeply reliant on floating-point intricacies).

The proper solution is to use big integers to exactly represent the given numbers.

**Comment by byt3h3ad** [comment]
@appgurueu thanks for the feedback! I have now used BigInt, and turns out there was indeed an error xD.

File: Project-Euler/Problem013.js [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 export const largeSum = (nums) => {
   const arr = nums.split('\n') // convert to array of strings
-  let sum = 0
+  let sum = 0n
   for (const item of arr) {
-    sum += parseInt(item)
+    sum += BigInt(item)
   } // calculate the sum
   const res = parseInt(sum.toString().substring(0, 11).replace('.', '')) // convert the sum to a string, get the first 10 digits and remove the decimal
```
Comment (appgurueu) [inline Project-Euler/Problem013.js]: You don't need to remove the decimal anymore

**Review by appgurueu — APPROVED** [review]

File: Project-Euler/Problem013.js (hunk 1 above)
Comment (byt3h3ad) [reply Project-Euler/Problem013.js]: thanks!

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**